### PR TITLE
Added "Julia language: a concise tutorial"

### PR DESCRIPTION
**Pull request by sylvaticus**
https://www.gitbook.com/book/sylvaticus/julia-language-a-concise-tutorial

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -1443,11 +1443,11 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 
 * [Introducing Julia wikibook](https://en.wikibooks.org/wiki/Introducing_Julia)
 * [Julia by Example](http://samuelcolvin.github.io/JuliaByExample) - Samuel Colvin (GitHub repo)
+* [Julia language: a concise tutorial](https://www.gitbook.com/book/sylvaticus/julia-language-a-concise-tutorial) - Antonello Lobianco (GitBook - PDF)
 * [Learn Julia in Y minutes](https://learnxinyminutes.com/docs/julia) - Leah Hanson
 * [Programming in Julia](http://quant-econ.net/jl/learning_julia.html) - Thomas J. Sargent and John Stachurski
 * [The Julia Express](http://bogumilkaminski.pl/files/julia_express.pdf) - Bogomił Kamiński (PDF)
 
-
```
Comment (eshellman) [inline free-programming-books.md]: Our linter, run by Travis, wants 2 spaces here.

File: free-programming-books.md [hunk 2]
```diff
@@ -1443,11 +1443,11 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 
 * [Introducing Julia wikibook](https://en.wikibooks.org/wiki/Introducing_Julia)
 * [Julia by Example](http://samuelcolvin.github.io/JuliaByExample) - Samuel Colvin (GitHub repo)
+* [Julia language: a concise tutorial](https://www.gitbook.com/book/sylvaticus/julia-language-a-concise-tutorial) - Antonello Lobianco (GitBook - PDF)
```
Comment (eshellman) [inline free-programming-books.md]: Great contribution. but you're not linking directly to pdf, so no notation is needed. Also, EPUB, MOBI, and HTML versions are also available

**Comment by sylvaticus** [comment]
Done removing the PDF tag and added the second line..

**Comment by eshellman** [comment]
Thanks!

**Pull request merged**